fairlearn/reductions/_exponentiated_gradient/_lagrangian.py

```diff
@@ -41,7 +41,7 @@ class _Lagrangian:
     def __init__(self, X, sensitive_features, y, estimator, constraints, eps, B, opt_lambda=True):
         self.X = X
         self.constraints = constraints
-        self.constraints.load_data(X, y, sensitive_features=sensitive_features)
+        self.constraints.load_data(X, y, eps=eps, sensitive_features=sensitive_features)
         self.obj = self.constraints.default_objective()
```

Comment:
we shouldn't need `eps`

Reply:
Removed it! Sorry we forgot it from our initial version of implementation, but it wasn't necessary